### ISSUE TITLE
Test all combinations of source/target for scoping

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/DSL#32]
**Is your feature request related to a problem? Please describe.**
Currently, only a subset of possible combinations of source/target for cross-references is checked.

Also, test the import system separately and only once.

### Tasks

- [ ] Test global scoping separately
- [ ] Think whether we can define tests for local scoping by a convention like we do for issue finder/formatter (linking done by ID):
```
// target 1
val »ref« = 1;

// source 1
»ref«;
```

**Comment by lars-reimann** [comment]
Superseded by #540.
